extensions/adobe/experience/offer-management/content/html-content.schema.json

```diff
@@ -0,0 +1,56 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id":
+    "https://ns.adobe.com/experience/offer-management/content/html-content",
+  "meta:extends": ["https://ns.adobe.com/experience/offer-management/representation-content"],
+  "meta:abstract": false,
+  "meta:extensible": true,
+  "type": "object",
+  "title": "HTML Content",
+  "description": "HTML content used in an offer representation",
+  "definitions": {
+    "html-content": {
+      "type": "object",
+      "title": "HTML Content",
+      "description": "HTML content used in an offer representation",
+      "properties": {
+        "@type": {
+          "type": "string",
+          "const": "https://ns.adobe.com/experience/offer-management/content/html-content"
+        },
+        "xdm:data": {
+          "type": "object",
+          "description": "Offer content data is a map, where the keys are two- or four-letter language tags. E.g. 'en', 'en-US'",
```

Comment:
We have this pattern elsewhere - please use that by reference instead of defining your own

Reply:
Please provide the schema component for that and we can evaluate if this fits. We discussed with Lars the usage of patternProperties as the correct way in XDM to express keys in open-ended key value maps.

Reply:
What I found is not a map but a `xdm:language` property in https://ns.adobe.com/xdm/content/componentized-page/#definitions/page. It is not an isolated pattern and most importantly it is not defined for the page component only for the page.
If you disagree, can you please provide the schema and definition where you think the pattern is made available for reuse?